CONTRIBUTING.md

```diff
@@ -1,126 +1,68 @@
-- `util` contains custom implementations of building blocks like an actor scheduler, buffer allocations, and metrics. Its parts are used in most of the other modules
-- `protocol` contains the SBE definition of the main message protocol
-- `bpmn-model` is a Java API for BPMN process definitions used for parsing etc.
-- `msgpack-*` is a custom msgpack implementation with extensions to evaluate json-path expressions on msgpack objects
-- `dispatcher` is a custom implementation of message passing between threads
-- `service-container` is a custom implementation to manage dependencies between different services
-- `logstreams` is an implementation of an append-only log backed by the filesystem
-- `transport` is our abstraction over network transports
-- `gateway` is the implementation of the gRPC gateway, using our SBE-based protocol to communicate with brokers
-- `gateway-protocol` contains the gRPC definitions for the Zeebe client-to-gateway protocol
-- `zb-db` is our RocksDB wrapper for state management
-- `engine`  is the implementation of the event stream processor
-- `broker` contains the Zeebe broker which is the server side of Zeebe
-- `client-java` contains the Java Zeebe client
-- `atomix` contains transport, membership, and consensus algorithms
-- `benchmark` contains utilities the team uses to run load tests
-- `exporters/elasticsearch-exporter` contains the official Elasticsearch exporter for Zeebe
-- `journal` contains the append-only log used by the consensus algorithm
-- `snapshots` abstracts how state snapshots (i.e. `zb-db`) are handled
-
-### Test Execution
-
-Tests can be executed via maven (`mvn verify`) or in your preferred IDE. The Zeebe Team uses mostly [Intellij IDEA](https://www.jetbrains.com/idea/), where we also [provide settings for](https://github.com/camunda/camunda/tree/main/.idea).
-
-> [!TIP]
-> To execute the tests quickly, run `mvn verify -Dquickly -DskipTests=false`.
-> The tests will be skipped when using `-Dquickly` without `-DskipTests=false`.
-
-#### Test Troubleshooting
-
-- If you encounter issues (like `java.lang.UnsatisfiedLinkError: failed to load the required native library`) while running the test StandaloneGatewaySecurityTest.shouldStartWithTlsEnabled take a look at https://github.com/camunda/camunda/issues/10488 to resolve it.
+# Contributing to Camunda
+
+We welcome new contributions. We take pride in maintaining and encouraging a friendly, welcoming, and collaborative community.
+
+Anyone is welcome to contribute to Camunda! The best way to get started is to choose an existing [issue](#starting-on-an-issue).
+
+Core code contributions must fit into the scope of Camunda products. The ultimate decision is up to the core Camunda team. If your vision does not fit in the core Camunda product, you may be directed to the [Camunda Community Hub](https://github.com/camunda-community-hub) and/or [Camunda Marketplace](https://marketplace.camunda.com/en-US/home) instead.
```

Comment:
Do we really feel this is necessary? It reads a bit deterring to me. It is kind of clear, that Camunda is the maintainer of this repo and will either accept or reject PRs. 

Maybe a simple note (not that prominent) that there is also a community hub would do it as well.

Reply:
Makes sense, I adjusted this section.